File: CS/DXExample.Module/Module.cs
using System;
using System.Collections.Generic;

using DevExpress.ExpressApp;
using System.Reflection;
using DevExpress.ExpressApp.SystemModule;
using DevExpress.ExpressApp.Model;


namespace DXExample.Module {
    public sealed partial class DXExampleModule : ModuleBase {
        public DXExampleModule() {
            InitializeComponent();
        }
        public override void ExtendModelInterfaces(ModelInterfaceExtenders extenders) {
            base.ExtendModelInterfaces(extenders);
            extenders.Add<IModelRootNavigationItems, IModelRootNavigationItemsEx>();
        }
    }
    public interface IModelRootNavigationItemsEx {
        bool AllowSelectFirstItem { get;set;}
    }
}

File: VB/DXExample.Module/Module.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic

Imports DevExpress.ExpressApp
Imports System.Reflection
Imports DevExpress.ExpressApp.SystemModule
Imports DevExpress.ExpressApp.Model


Namespace DXExample.Module
	Public NotInheritable Partial Class DXExampleModule
		Inherits ModuleBase
		Public Sub New()
			InitializeComponent()
		End Sub
		Public Overrides Sub ExtendModelInterfaces(ByVal extenders As ModelInterfaceExtenders)
			MyBase.ExtendModelInterfaces(extenders)
			extenders.Add(Of IModelRootNavigationItems, IModelRootNavigationItemsEx)()
		End Sub
	End Class
	Public Interface IModelRootNavigationItemsEx
		Property AllowSelectFirstItem() As Boolean
	End Interface
End Namespace
